src/test/rustdoc-ui/issue-79494.rs

```diff
@@ -0,0 +1,9 @@
+#![feature(const_transmute)]
+
+fn main() {
+    match &b""[..] {
+        ZST => {}
+    }
+}
```

Comment:
Based on the issue thread, it sounds like this function isn't needed? The transmute is enough?

Reply:
I removed the function!